types/result.go

```diff
@@ -90,7 +91,7 @@ func NewResponseResultTx(res *ctypes.ResultTx, tx Tx) TxResponse {
 		TxHash:    res.Hash.String(),
 		Height:    res.Height,
 		Code:      res.TxResult.Code,
-		Data:      res.TxResult.Data,
+		Data:      strings.ToUpper(hex.EncodeToString(res.TxResult.Data)),
```

Comment:
can't we use `res.TxResult.Data.String()`  instead of `hex.EncodeToString()`?

Reply:
No, `Data` is a raw `[]byte` type, not a Tendermint `HexBytes` type.